src/server/json_family.cc

```diff
@@ -1447,10 +1447,16 @@ io::Result<JsonPathV2, string> ParsePathV2(string_view path) {
   }
 
   if (absl::GetFlag(FLAGS_jsonpathv2)) {
-    return json::ParsePath(path);
+    auto path_result = json::ParsePath(path);
+    if (!path_result) {
+      LOG(WARNING) << "Invalid Json path: " << path << ' ' << path_result.error() << std::endl;
```

Comment:
It is not good to use LOG(WARNING) in commands path for wrong input coming from user, as this can explode the log if user sends many of this commands. We use VLOG(2) for this cases.

Reply:
@BagritsevichStepan maybe it's also worth replacing `LOG(WARNING)` with `VLOG` on doc_accessor as well ? Apologies for the back an forth here I should have noticed the hot path and log from the beginning

Reply:
no strong preference :)

Reply:
I agree that we should not usually log based on a user input - it's a security risk. However, VLOG(2) is pretty verbose, it may log stuff per request . I think `VLOG(1) ` can work well for such situations, where we may want to  set level 1 in production just to see these corner cases without the danger of exploding logs because of other VLOG(2) printings.